tests/unit/src/test/scala/tests/CodeLensLspSuite.scala

```diff
@@ -357,56 +357,76 @@ class CodeLensLspSuite extends BaseCodeLensLspSuite("codeLenses") {
         |""".stripMargin
   )
 
-  test("run-shell-command") {
-
-    def runFromCommand(cmd: Command) = {
+  private def runFromCommand(cmd: Command) = {
+    cmd.getArguments().asScala.toList match {
+      case (params: DebugSessionParams) :: _ =>
+        params.getData() match {
+          case obj: JsonObject =>
+            val cmd = obj
+              .get("shellCommand")
+              .getAsString()
+              // need to remove all escapes since ShellRunner does escaping itself
+              .replace("\\\"", "")
+              .replace("\"\\", "\\")
+              // when testing on Windows Program Files is problematic when splitting into list
+              .replace("Program Files", "ProgramFiles")
```

Comment:
I am so very sorry about this :O

Reply:
Woof

Reply:
The other option would be to find a process runner that takes in a whole string, but that seems even more work :/